MbDotNet/MountebankRequestProxy.cs

```diff
@@ -121,6 +123,24 @@ private HttpResponseMessage ExecutePost(string resource, string json)
             }
         }
 
+        private void HandleDynamicPort(HttpResponseMessage response, Imposter imposter)
+        {
+            if (imposter.Port == null)
+            {
+                try
+                {
+                    var content = response.Content.ReadAsStringAsync().Result;
+                    Match match = new Regex(".*\"port\": (\\d+),", RegexOptions.Multiline).Match(content);

```

Comment:
I also considered adding the Port property to the RetrievedImposter and getting it that way, but that detail isn't really important for imposter verification which is the intended purpose of that model.  I could even add a new model type that only contains the port that is used here.  That might arguably be cleaner.